.github/workflows/int-and-connected-test-run.yml

```diff
@@ -68,23 +117,18 @@ jobs:
         run: |
           # write vault token
           base64 --decode <<< ${B64_RBS_APPLICATION_CREDENTIALS} > ${RBS_CLIENT_CREDENTIAL_FILE_PATH}
-      - name: "Run Connected test via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.62.0
+      - name: "Run connected tests via Gradle"
+        uses: broadinstitute/datarepo-actions/actions/main@0.63.0
         with:
           actions_subcommand: 'gradleinttest'
           pgport: ${{ job.services.postgres.ports[5432] }}
           test_to_run: 'testConnected'
           role_id: ${{ secrets.ROLE_ID }}
           secret_id: ${{ secrets.SECRET_ID }}
-      - name: "Archive connected junit test reports"
-        if: always()
-        uses: actions/upload-artifact@v2
-        with:
-          name: junit-test-report-connected
-          path: build/reports
-          retention-days: 10
   deploy_test_integration:
-    timeout-minutes: 210
+    name: "Run integration and smoke tests"
+    timeout-minutes: 300
```

Comment:
Want to double-check, are we intending to keep this high timeout?

Reply:
Yeah, I figure I'll leave it a bit higher since that way we can actually get the scan results to debug tests